legacy/web/src/test/java/org/jboss/as/web/test/WebMigrateTestCase.java

```diff
@@ -302,6 +303,10 @@ protected RunningMode getRunningMode() {
             return RunningMode.ADMIN_ONLY;
         }
 
+        @Override
+        protected ProcessType getProcessType() {
+            return ProcessType.SELF_CONTAINED;
```

Comment:
maybe host controller is more appropriate, but cli self contained server could work as well.